affected:
- package:
    ecosystem: PyPI
    name: Plone
  ranges:
  - events:
    - introduced: 3.3.2
    - fixed: 3.3.6
    type: ECOSYSTEM
aliases:
- CVE-2011-2528
details: 'Unspecified vulnerability in (1) Zope 2.12.x before 2.12.19 and 2.13.x before
  2.13.8, as used in Plone 4.x and other products, and (2) PloneHotfix20110720 for
  Plone 3.x allows attackers to gain privileges via unspecified vectors, related to
  a "highly serious vulnerability." NOTE: this vulnerability exists because of an
  incorrect fix for CVE-2011-0720.'
id: PYSEC-2011-25
modified: '2024-11-25T18:35:18.357593Z'
published: '2011-07-19T20:55:00Z'
references:
- type: FIX
  url: http://plone.org/products/plone-hotfix/releases/20110622
- type: ADVISORY
  url: http://plone.org/products/plone-hotfix/releases/20110622
- type: FIX
  url: https://bugzilla.redhat.com/show_bug.cgi?id=718824
- type: FIX
  url: http://www.openwall.com/lists/oss-security/2011/07/12/9
- type: FIX
  url: http://plone.org/products/plone/security/advisories/20110622
- type: ADVISORY
  url: http://plone.org/products/plone/security/advisories/20110622
- type: FIX
  url: https://mail.zope.org/pipermail/zope-announce/2011-June/002260.html
- type: ADVISORY
  url: http://secunia.com/advisories/45056
- type: ADVISORY
  url: http://secunia.com/advisories/45111
- type: FIX
  url: http://www.openwall.com/lists/oss-security/2011/07/04/6
